demo/scripts/controlsV2/sidePane/snapshot/SnapshotPane.tsx

```diff
@@ -73,6 +73,7 @@ export class SnapshotPane extends React.Component<SnapshotPaneProps, SnapshotPan
             entityStates,
             isDarkMode,
             selection,
+            logicalRootPath: [],
```

Comment:
I didn't add functionality to show this in the pane, please let me know if I should

Reply:
You may need to add a text box to show/add the selection path. But this is low pri compared to the main feature.